two_factor/views/core.py

```diff
@@ -548,6 +548,7 @@ def done(self, form_list, **kwargs):
         # PhoneNumberForm / YubiKeyDeviceForm / EmailForm / WebauthnDeviceValidationForm
         elif method.code in ('call', 'sms', 'yubikey', 'email', 'webauthn'):
             device = self.get_device()
+            device.confirmed = True
```

Comment:
There are no tests that cover this change

Reply:
I assumed that default setup flow already covers it since it uses confirmed property as default, but added a new test to cover that scenario as well to separate the concerns. Thanks